cmd/kubeadm/app/cmd/phases/join/preflight.go

```diff
@@ -87,7 +86,7 @@ func runPreflight(c workflow.RunData) error {
 	fmt.Println("[preflight] Running pre-flight checks")
 
 	// Start with general checks
-	klog.V(1).Infoln("[preflight] Running general checks")
+	fmt.Println("[preflight] Running general checks")
```

Comment:
we should keep the original V(1) here adds more detail to the line `fmt.Println("[preflight] Running pre-flight checks")
` line above.
[1]